packages/node/src/integrations/tracing/fastify-v3/enums/AttributeNames.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
For the files we vendored in from OTEL, we should also link to the git sha we used of the versioned files.